app/controllers/events/view/index.js

```diff
@@ -1,51 +1,76 @@
 import Controller from '@ember/controller';
+import { computed, action } from '@ember/object';
+import EmberTableControllerMixin from 'open-event-frontend/mixins/ember-table-controller';
 
-export default Controller.extend({
-  sponsorsColumns: [
-    {
-      propertyName   : 'logo-url',
-      template       : 'components/ui-table/cell/cell-image',
-      title          : 'Logo',
-      disableSorting : true
-    },
-    {
-      propertyName : 'name',
-      title        : 'Name'
-    },
-    {
-      propertyName   : 'type',
-      template       : 'components/ui-table/cell/cell-sponsor-sanitize',
-      title          : 'Type',
-      disableSorting : true
-    },
-    {
-      propertyName : 'level',
-      template     : 'components/ui-table/cell/cell-sponsor-sanitize',
-      title        : 'Level'
-    },
-    {
-      title          : 'Options',
-      template       : 'components/ui-table/cell/cell-sponsor-options',
-      disableSorting : true
-    }
-  ],
-  actions: {
-    deleteSponsor(sponsor) {
-      this.set('isLoading', true);
-      sponsor.destroyRecord()
-        .then(() => {
-          this.notify.success(this.l10n.t('Sponsor has been deleted successfully.'));
-          this.get('model.sponsors').removeObject(sponsor);
-        })
-        .catch(() => {
-          this.notify.error(this.l10n.t('An unexpected error has occurred.'));
-        })
-        .finally(() => {
-          this.set('isLoading', false);
-        });
-    },
-    editSponsor() {
-      this.transitionToRoute('events.view.edit.sponsors');
-    }
+export default class extends Controller.extend(EmberTableControllerMixin) {
+  per_page = 100;
+
+  sort_dir = 'ASC';
+
+  @computed()
+  get sponsorsColumns() {
+    return [
+      {
+        name          : 'Logo',
+        valuePath     : 'logo-url',
+        cellComponent : 'ui-table/cell/cell-image'
+      },
+      {
+        name            : 'Name',
+        valuePath       : 'name',
+        headerComponent : 'tables/headers/sort',
+        isSortable      : true
+
+      },
+      {
+        name            : 'Type',
+        valuePath       : 'type',
+        cellComponent   : 'ui-table/cell/cell-sponsor-sanitize',
+        extraValuePaths : ['type'],
+        options         : {
+          PropName: 'type'
```

Comment:
why?

Reply:
Since for `Type` and `Level` we use a single cell component, I used it to compare when to use `level value path` and when to use `type value path`

Reply:
you can just write 
```hbs
{{#if record}}
 {{record}}
{{/record}}
```
Why do you want to differentiate anyway?

Reply:
Done